entry.go

```diff
@@ -609,11 +643,19 @@ func StartReverse(config *EntryConfiguration, wallet *nkn.Wallet) error {
 					}
 
 					encryptedConn, connMetadata, err := te.wrapConn(tcpConn, nil, nil)
+					encryptKeys = te.encryptKeys
 					if err != nil {
 						return fmt.Errorf("wrap conn error: %v", err)
 					}
 
 					defer Close(encryptedConn)
+					defer delete(te.encryptKeys, string(append(connMetadata.PublicKey, connMetadata.Nonce...)))
+					defer delete(encryptKeys, string(append(connMetadata.PublicKey, connMetadata.Nonce...)))
```

Comment:
You have L646 to make `encryptKeys` and `te.encryptKeys` pointing to the same map